Makefile

```diff
@@ -98,11 +92,21 @@ lint_module: $(FAILLINT) $(GOLANGCI_LINT) $(MISSPELL)
 	@cd $(DIR) && $(GOLANGCI_LINT) run
 	@./scripts/git-tree.sh
 
-# TODO(bwplotka): Move to buf.
-$(PROTOC):
+
+# For protoc naming matters.
+PROTOC_GEN_GO_CURRENT := $(TMP_GOPATH)/protoc-gen-go
+PROTOC_GEN_GO_GRPC_CURRENT := $(TMP_GOPATH)/protoc-gen-go-grpc
+PROTO_TEST_DIR := testing/testpb/v1
+
+.PHONY: proto
+proto: ## Generate testing protobufs
+proto: $(BUF) $(PROTOC_GEN_GO) $(PROTOC_GEN_GO_GRPC) $(PROTO_TEST_DIR)/test.proto
 	@mkdir -p $(TMP_GOPATH)
-	@echo ">> fetching protoc@${PROTOC_VERSION}"
-	@PROTOC_VERSION="$(PROTOC_VERSION)" TMP_GOPATH="$(TMP_GOPATH)" scripts/installprotoc.sh
-	@echo ">> installing protoc@${PROTOC_VERSION}"
-	@mv -- "$(TMP_GOPATH)/bin/protoc" "$(GOBIN)/protoc-$(PROTOC_VERSION)"
-	@echo ">> produced $(GOBIN)/protoc-$(PROTOC_VERSION)"
+	@cp $(PROTOC_GEN_GO) $(PROTOC_GEN_GO_CURRENT)
+	@cp $(PROTOC_GEN_GO_GRPC) $(PROTOC_GEN_GO_GRPC_CURRENT)
+	@echo ">> generating $(PROTO_TEST_DIR)"
+	@PATH=$(GOBIN):$(TMP_GOPATH) $(BUF) protoc \
+		-I $(PROTO_TEST_DIR) \
+		--go_out=$(PROTO_TEST_DIR)/../ \
+		--go-grpc_out=$(PROTO_TEST_DIR)/../ \
+	    $(PROTO_TEST_DIR)/*.proto
```

Comment:
This is awesome, next step would be to just use `buf generate`: https://docs.buf.build/tour-6/. If you're feeling keen it should be possible to do here already with `buf.gen.yaml`:

```yaml
version: v1beta1
plugins:
  - name: go
    out: testing/testpb/v1
  - name: go-grpc
    out: testing/testpb/v1
```

You may want `opt: paths=source_relative` if the files aren't going in the right place.

Reply:
uuu nice, let's try it (or in sep PR)